public/css/index.css

```diff
@@ -85,3 +85,82 @@
     justify-content: center;
     overflow-x: hidden;
 }
+
+/*........... BLOG SECTION.......... */
+#post-container {
+  padding: 0 25px;
+  max-width: 1400px;
+  margin: 0 auto;
+
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: 1fr 1fr 1fr;
+  gap: 20px 20px;
+}
+
+.odd-row.large-post {
+  grid-column: 1 / span 2;
+}
+
+.even-row.large-post {
+  grid-column: 2 / span 2;
+}
+
+#post-container div {
+  height : 400px;
```

Comment:
This conflicts with the blog post subtitles. The subtitles become 400px tall and create blocks of empty space in the index page. 
```html
<div class="subtitle">November 1, 2022</div>
```
Perhaps change this to a `<p>` element or create a separate CSS statement for the subtitle div class.